wazo_sysconfd/controller.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, config: dict):
                 'status_aggregator': self.status_aggregator,
             },
         )
+        logger.debug('Loaded plugins:\n%s', plugin_manager.names())
```

Comment:
Out of curiosity, what is the plus value of this logger? `plugin_helpers.load` already log enabled plugins